README.md

```diff
@@ -17,37 +17,39 @@ The official SurrealDB library for Golang.
 go get github.com/surrealdb/surrealdb.go
 ```
 
-### Usage
+### Quick Start
 
 ```go
 package main
-
 import (
-    "fmt"
-    "github.com/surrealdb/surrealdb.go"
+	"github.com/surrealdb/surrealdb.go"
+
 )
 
+type User struct {
+	ID      string `json:"id,omitempty"`
+	Name    string `json:"name"`
+	Surname string `json:"surname"`
+}
+
 func main() {
-	// Connect to SurrealDB
```

Comment:
I can't apply a suggestion to a deleted line, but could we revert this please? We do actually have people who aren't familiar with connecting to databases, so it would be good to have the comment

Reply:
Yep absolutely. The other thing is do you think it would make sense to have a line on how to install and start SurrealDB on your local system or atleast point to documentation which can help them figure it out.

Reply:
Yes, a link is better thank you!

Reply:
This is done above.